diff --git a/node_modules/knex/lib/dialects/postgres/index.js b/node_modules/knex/lib/dialects/postgres/index.js
index 6b57718..671d052 100644
--- a/node_modules/knex/lib/dialects/postgres/index.js
+++ b/node_modules/knex/lib/dialects/postgres/index.js
@@ -124,10 +124,7 @@ class Client_PG extends Client {
   // checking on the database.
   checkVersion(connection) {
     return new Promise((resolve, reject) => {
-      connection.query('select version();', (err, resp) => {
-        if (err) return reject(err);
-        resolve(this._parseVersion(resp.rows[0].version));
-      });
+      resolve(22);
     });
   }
 
@@ -168,7 +165,7 @@ class Client_PG extends Client {
           .join(', ')}]`;
         this.logger.warn(
           `Detected comma in searchPath "${path}".` +
-            `If you are trying to specify multiple schemas, use Array syntax: ${arraySyntax}`
+          `If you are trying to specify multiple schemas, use Array syntax: ${arraySyntax}`
         );
       }
       path = [path];
